adserver/templates/adserver/dashboard.html

```diff
@@ -14,14 +14,26 @@
 
 {% block content_container %}
 
+  {% if request.user.is_staff %}
+    <h2>{% trans 'Staff Reports' %}</h2>
+    <p>
+      {% trans 'Reports only visible to staff users.' %}
+    </p>
+    <ul>
+        <li><a href="{% url 'all_advertisers_report' %}">{% trans 'All advertiser report' %}</a></li>
+        <li><a href="{% url 'all_publishers_report' %}">{% trans 'All publisher report' %}</a></li>
+        <li><a href="{% url 'all_keyword_report' %}">{% trans 'All keyword report' %}</a></li>
+        <li><a href="{% url 'all_geo_report' %}">{% trans 'All geo report' %}</a></li>
+        <li><a href="{% url 'publisher_uplift_report' %}">{% trans 'All uplift report' %}</a></li>
```

Comment:
I know the other reports have "all" in front of them but this one doesn't in the actual report template.

```suggestion
        <li><a href="{% url 'publisher_uplift_report' %}">{% trans 'Uplift report' %}</a></li>
```